src/pages/home/platform-carousel/_mobile-platform-carousel.tsx

```diff
@@ -103,50 +103,64 @@ const MobilePlatformCarousel = () => {
     const { is_row, is_eu, is_uk } = getCountryRule()
     const images = useStaticQuery(image_query)
 
+    const carouselData = () => {
```

Comment:
You can write this like this. (It would be better in performance)

`
const carouselData = {
     is_row: platform_details_cr,
     is_eu: platform_details_eu,
     is_uk: platform_details_uk
}
`